### ISSUE TITLE
About reproduction on SNLI

### ISSUE THREAD

**Issue by iboxty** [facebookresearch/InferSent#92]
Hi,

I run the train_nli.py with the default parameters. But I end up with getting about 80% and 79% accuracies on the dev/test set, respectively. I run the code on one single GTX 1080 Ti GPU and get nearly the same results in two dependent runs.

What parameters I should use to get the proper results? or is there any potential reason to cause the failure of reproduction? 
BTW, I use the glove pre-trained embeddings (and the default parameters in train_nli.py).

Thanks! Looking forward to your reply!

**Comment by yrchen92** [comment]
I also have some difficulties for reproduction and get the same 80%/80%/79% accuracies on the train/dev/test set.
